src/aggregators/aggregators.jl

```diff
@@ -79,6 +79,10 @@ systems with many species and many channels, Journal of Physical Chemistry A,
 """
 struct NRM <: AbstractAggregatorAlgorithm end
 
+"""
+citation
```

Comment:
missing

Reply:
@Vilin97 The original paper is: Elf, Johan and Ehrenberg, M, Spontaneous separation of bi-stable biochemical systems into spatial domains of opposite phases,Systems Biology, 2004 vol. 1(2) pp. 230-236

Reply:
Thank you. Added